### PR TITLE
View longterm entries

### PR DESCRIPTION
**Pull request by nmohamed**
woohoo #108
I'd like to add on to this, but the way we set dates for longterm entries is weird. you can make multiple entries for the same month on different dates for example

### CODE REVIEW

**Comment by davidabrahams** [comment]
Agreed we need to fix longterm entry dates. Can someone take charge on that issue?

Otherwise, this is good to merge!

File: sass/stylesheets/header.sass [hunk 1]
```diff
@@ -2,6 +2,8 @@
   background: #41afa5
   border: none
   margin-bottom: 0px
+  border-radius: 0px
```
Comment (davidabrahams) [inline sass/stylesheets/header.sass:5]: is this necessary even though border=none

**Comment by davidabrahams** [comment]
I went ahead and dealt with merge conflicts. I'm 93% sure I didn't break functionality, but @nmohamed you should probably test. If I did, feel free to just revert my commit and do it yourself

**Comment by davidabrahams** [comment]
:bus:

File: sass/stylesheets/header.sass (hunk 1 above)
Comment (nmohamed) [inline sass/stylesheets/header.sass:5]: the header has rounded corners, so that's why i did this

**Comment by BrennaManning** [comment]
:+1:

**Comment by aconsilvio** [comment]
🚌

**Pull request merged**